### ISSUE TITLE
Link for semantic/gesture annotations is broken

### ISSUE THREAD

**Issue by jennhu** [PantoMatrix/PantoMatrix#84]
Hello, thanks for releasing your code and dataset!

I am interested in the annotations of gesture types (no gestures (0), beat gestures (1), low-middle-high quaility deictic gestures (2-3-4), iconic gestures (5-6-7), metaphoric gestures (8-9-10)). However, the link to annotation tools appears to broken. The [README](https://github.com/PantoMatrix/BEAT#other-scripts-and-avatars) points to this broken Google Drive link: https://drive.google.com/file/d/1HuVoTWgDXKX0M3yfei1jY3Pr7ZMF5k3G/view?usp=share_link

Would you be able to update the link or share the annotations another way? Thank you!

**Comment by H-Liu1997** [comment]
Thanks, I already shared the tools, please let me know if you have other concerns.

**Comment by KevinGoodman** [comment]
@H-Liu1997 It seems that the above link is still broken, could you please provide a new link ?

**Comment by H-Liu1997** [comment]
@KevinGoodman yes. https://huggingface.co/datasets/H-Liu1997/BEAT2/tree/main/beat_english_v2.0.0/sem

**Comment by GuangtaoLyu** [comment]
> [@KevinGoodman](https://github.com/KevinGoodman) yes. https://huggingface.co/datasets/H-Liu1997/BEAT2/tree/main/beat_english_v2.0.0/sem

Are there corresponding sem annotations for chinese, japanese and english?
